built-in_analyses.js

```diff
@@ -15,14 +15,18 @@ function my_built_in_analysis(){
   /* Step 4: (optional) If you want to annotate your tableaux with tones,
    * uncomment this block: */
   /*
-  var toneButtons = document.getElementsByName("toneOptions");
+  var toneCheckbox = document.getElementById("annotatedWithTones");
+  toneCheckbox.setAttribute("checked", "checked");
+  var toneButtons = toneCheckbox.parentNode.parentNode.getElementsByTagName("input");
   for(var x = 0; x < toneButtons.length; x++){
+    toneButtons[x].parentNode.setAttribute("style", "display: table-cell");
     if(toneButtons[x].value="addIrishTones_Elfner"){
```

Comment:
This needs to be a "===". You were resetting the value of all the other buttons to "addIrishTones_Elfner"! This is what was causing problems below.